modules/api/src/main/java/org/apache/ignite/sql/SqlBatchException.java

```diff
@@ -30,6 +29,9 @@
  *
  */
 public class SqlBatchException extends SqlException {
+    /** Empty array of long. */
+    public static final long[] LONG_EMPTY_ARRAY = new long[0];
```

Comment:
Are you sure it's supposed to be public?

Reply:
You are right, it should be private. Fixed